docs/docs-content/clusters/cluster-management/certificate-management.md

```diff
@@ -13,12 +13,11 @@ recommend you review the
 [PKI certificates and requirement](https://kubernetes.io/docs/setup/best-practices/certificates) Kubernetes
 documentation to learn more about the auto-generated certificates and to better understand their purpose.
```

Comment:
somewhere in this doc we need to explain how automatic renewals behave. 30 days before expiry, etc.